### ISSUE TITLE
Input() doesn't work

### ISSUE THREAD

**Issue by Marko2155** [vbe0201/3DS.py#23]
I want to make a Python interpreter for the 3DS, but the "Function not implemented" error is stopping me. Maybe add a check for input() and if so pop up the 3DS keyboard?

**Comment by the-duck2010** [comment]
Yeah that is an issue. My suggestion is to have a custom keyboard on the lower screen, custom so that you can have arrow keys for selection in CLI menus.
